react-js-buy/src/App.js

```diff
@@ -65,7 +60,7 @@ class App extends Component {
     });
   }
 
-  removeLineItemInCart(lineItemId) {
+  removeLineItemInCart = (lineItemId) => {
```

Comment:
Please don't make unnecessary changes to code style. We use function declarations for consistency. Arrow functions are fine for anonymous functions, but named functions should use a function declaration.